applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -32,6 +32,10 @@ static cs47l63_t cs47l63_driver;
 
 /**@brief Write to multiple registers in CS47L63
  */
+uint32_t hw_codec_volume_get()
+{
+	return prev_volume_reg_val;
```

Comment:
I need to look into this, but the only way to "properly" get the volume is to read it from the HW codec itself. Discuss, and see if we have another option using VCS.

Reply:
I guess this value will also be available on zbus when the "mothership" PR is merged?